fiftyone/utils/sam2.py

```diff
@@ -257,23 +261,24 @@ class SegmentAnything2VideoModel(fom.SamplesMixin, fom.Model):
     """
 
     def __init__(self, config):
-        self._fields = {}
```

Comment:
@brimoor Why did you get rid of this line? I assume it was by mistake?

Reply:
oh I didn't see it used anywhere but I guess it is!